### DISCUSSION TITLE
404 Not Found: / i get this error in the command line for a django project

### DISCUSSION THREAD

**Discussion by ARK003** [community/community#29060] [Other Feature Feedback, Questions, & Ideas]
Hello ,
i would really appreciate if anyone could help i get an error 404 page not found and this in the command line
views.py file
from django.shortcuts import render

def home (request):
  return render(request, 'index.html')

urls.py file
from django import views
from django.contrib import admin
from django.urls import path
from . import views

urlpatterns = [
    path ('admin/', admin.site.urls),
    path ('home/',views.home,name = 'home')
    
]

**Comment by jfullstackdev** [comment]
you don't have any public repo for GitHub Pages, if that is what you are referring too 

![image](https://user-images.githubusercontent.com/47092464/184068804-ae640d2b-dd4a-4a01-83ef-7429101e19bd.png)
